bundles/org.openhab.binding.modbus.studer/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<!-- BSP Thing Type -->
+	<thing-type id="bsp">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="serial"/>
+		</supported-bridge-type-refs>
+		<label>BSP Studer</label>
+		<description>Thing for Studer BSP Device</description>
+		<channels>
+			<channel id="batteryVoltage" typeId="Ubat"/>
+			<channel id="batteryCurrent" typeId="Ibat"/>
+			<channel id="power" typeId="Pbat"/>
+			<channel id="stateOfCharge" typeId="SOC"/>
+			<channel id="batteryTemperature" typeId="Tbat"/>
+		</channels>
+		<config-description>
+			<parameter name="slaveAddress" type="integer" min="60" max="61">
+				<label>Slave Address</label>
+				<description>Slave address of BSP device</description>
+				<default>60</default>
+			</parameter>
+			<parameter name="refreshSec" type="integer" min="1" unit="s">
```

Comment:
The thing definition `refresh` should also be updated:
```suggestion
			<parameter name="refresh" type="integer" min="1" unit="s">
```
Same comment for other things.